Vagrantfile

```diff
@@ -0,0 +1,62 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+ENV['VAGRANT_DEFAULT_PROVIDER'] ||= "virtualbox"
+DOCKER_PORT=2376
+UNICORN_PORT=3000
+VAGRANTFILE_API_VERSION = "2"
+
+$docker_setup=<<CONF
+cat > /etc/default/docker << 'EOF'
+DOCKER_OPTS="-H 0.0.0.0:#{DOCKER_PORT} -H unix:///var/run/docker.sock"
+EOF
+service docker restart
+CONF
+
+unless Vagrant.has_plugin?("vagrant-cachier")
+  puts "WARNING: vagrant-cachier plugin is not installed! It really speeds this up..."
+  puts "         Install using 'vagrant plugin install vagrant-cachier'"
+end
+
+unless Vagrant.has_plugin?("vagrant-persistent-storage")
+  puts "WARNING: vagrant-persistent-storage not installed! It really speeds up docker image mgmt"
+  puts "         Install using 'vagrant plugin install vagrant-persistent-storage'"
+end
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  config.vm.box = "ubuntu/trusty64"
+  config.vm.hostname = "prison-visits"
+  config.vm.network "forwarded_port", guest: DOCKER_PORT, host: DOCKER_PORT
+  config.vm.network "forwarded_port", guest: UNICORN_PORT, host: UNICORN_PORT
+
+  # Set up SSH agent forwarding.
+  config.ssh.forward_agent = true
+
+  if Vagrant.has_plugin?("vagrant-persistent-storage")
```

Comment:
Hmmm this plugin *only* works for Vbox. I guess the question is what does it do on vmware (does it fail to run or just have no effect)?